### ISSUE TITLE
[Feature request] Support to map YAW in wiinusoft

### ISSUE THREAD

**Issue by Neelgodly** [KeyPuncher/WiinUPro#95]
Non motionplus remotes probably can't do this but it would be awesome if we could map yaw if we have a motionplus attached. So far, only pitch and roll can be mapped.

**Comment by KeyPuncher** [comment]
It is on the roadmap for adding gyro support. It's fairly complicated with trying to retaining accuracy and reducing drift. No realistic idea of when I can actually start tackling it outside of just my head but it's planned (at least I guess).

**Comment by Neelgodly** [comment]
> It is on the roadmap for adding gyro support. It's fairly complicated with trying to retaining accuracy and reducing drift. No realistic idea of when I can actually start tackling it outside of just my head but it's planned (at least I guess).

Thats awesome, another feature that would be cool would be to map the "swing" of the accel rather than just the tilt. This would enable games like tennis to be played over parsec

**Comment by KeyPuncher** [comment]
> another feature that would be cool would be to map the "swing" of the accel rather than just the tilt. This would enable games like tennis to be played over parsec

I agree, that is also on my to do list. For my deep learning course last Fall I made a machine learning model to detect swing directions from accelerometer values. It did pretty good, with some tweaks and perhaps even feeding it gyro values it could be good enough to use in the application.
